app/assets/stylesheets/basic.scss

```diff
@@ -0,0 +1,36 @@
+
+
+.background {
+  display: flex;
+  flex-flow: column;
+  height: 100%;
+  background-color: #ADD8E6;
+  // background: url("http://www.wallpaperup.com/uploads/wallpapers/2012/09/26/16626/8155b41ae43140299c342079a2c134dd.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  overflow: auto;
+}
+
+/************** FIXES *******************************/
+
+*, *:before, *:after {
+  -moz-box-sizing: border-box;
```

Comment:
Each selector in a comma sequence should be on its own line<br>Avoid vendor prefixes.